tests/test_middleware_interface.py

```diff
@@ -145,26 +158,48 @@ def test_prep_butler(self):
         loaded_shards = list(self.butler.registry.queryDataIds("htm7",
                                                                datasets="gaia",
                                                                collections="refcats"))
+
+        # Check that the right skymap is in the chained output collection.
+        self.butler.datasetExists("skyMap",
+                                  # TODO: we shouldn't need skymap here?
+                                  skymap="deepCoadd_skyMap",
+                                  collections=self.interface.output_collection)
+
         # These shards were identified by plotting the objects in each shard
         # on-sky and overplotting the detector corners.
         # TODO DM-34112: check these shards again with some plots, once I've
         # determined whether ci_hits2015 actually has enough shards.
-        expected_shards = [157401, 157405, 157407]
+        expected_shards = [157394, 157401, 157405]
         self.assertEqual(expected_shards, [x['htm7'] for x in loaded_shards])
-        # check that we got appropriate calibs.
+        # Check that the right calibs are in the chained output collection.
         try:
             self.butler.datasetExists('cpBias', detector=56, instrument='DECam',
                                       collections="DECam/calib/20150218T000000Z")
+            # TODO: Have to use the exact calib collection, because we don't
+            # have validity ranges for data.
```

Comment:
```suggestion
            # TODO: Have to use the exact run, because we can't
            # query by validity range.
```
Please don't use "calib collection" to refer to something that's not a calibration collection; it's confusing. Same for the comment below.